2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,43 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+let favoriteRecipe1 = {
+  title: "Mole",
+  servings: 2,
+  ingredients: ["cinnamon", "cumin", "cocoa"]
+};
+let favoriteRecipe2 = {
+  title: "Lentil Soup",
+  servings: 3,
+  ingredients: ["Red Lentil", "Tomato paste", "Carrot", "Onion"]
+};
+let favoriteRecipe3 = {
+  title: "Oatmeal",
+  servings: 4,
+  ingredients: ["oats", "sugar", "cinnamon", "milk", "salt"]
+};
+let favoriteRecipe4 = {
+  title: "Pasta",
+  servings: 4,
+  ingredients: ["pasta", "oil", "cheese", "milk", "salt"]
+};
+let favoriteRecipe5 = {
+  title: "Pancakes",
+  servings: 4,
+  ingredients: ["eggs","flour","coconut oil","baking powder","sugar","milk"]
+};
+
+const favoriteRecipe = [
+  favoriteRecipe1,
+  favoriteRecipe2,
+  favoriteRecipe3,
+  favoriteRecipe4,
+  favoriteRecipe5,
+];
+for (let i = 0; i < 5; i++) {
+  console.log(favoriteRecipe[i].title);
+  console.log("Serves: " + favoriteRecipe[i].servings);
+  console.log("Ingredients:");
+  favoriteRecipe[i].ingredients.forEach((item) => console.log(item));
+}
```

Comment:
nice :) , you can use favorite recipe.length in your for loop, what if someone adds a recipe to your array?